src/handlers/viewSubmitInteraction.ts

```diff
@@ -0,0 +1,66 @@
+import { IModify, IRead } from '@rocket.chat/apps-engine/definition/accessors';
+import { UIKitViewSubmitInteractionContext } from '@rocket.chat/apps-engine/definition/uikit';
+
+import { TodoistApp } from '../../TodoistApp';
+import { ModalsEnum } from '../enums/Modals';
+import { createTask } from '../lib/create/createTask';
+import { deleteComment, deleteLabel, deleteSection, deleteTask } from '../lib/delete/deleteItem';
+
+export class ExecuteViewSubmitHandler {
+  constructor(
+    private readonly app: TodoistApp,
+    private readonly read: IRead,
+    private readonly modify: IModify
+  ) {}
+
+  public async run(context: UIKitViewSubmitInteractionContext) {
+    const data = context.getInteractionData();
+    let { view, room } = data;
+    const elements = view.id.split('#');
+    const viewId = elements[0];
+    const roomId = elements[1];
+    if (!room) room = await this.read.getRoomReader().getById(roomId);
+    if (!room) return context.getInteractionResponder().errorResponse();
+    try {
+      switch (viewId) {
```

Comment:
my advice, log what came from viewId and roomId, so you can visibility when things stop working